test/api/todos.coffee

```diff
@@ -97,6 +97,36 @@ describe "Todos", ->
           expect(todo.value).to.equal 0
           done()
 
+      it "Creates a completed a todo when using up url", (done) ->
+        upTodo = undefined
+        request.post(baseURL + "/user/tasks/withUp/up").send(
+            type: "todo"
+            text: "testTitle"
+            complete: true
+            description: "testDesc"
+        ).end (res) ->
+          expectCode res, 200
+          request.get(baseURL + "/user/tasks/withUp")
+          .send().end (res) ->
+            upTodo = res.body
+            expect(upTodo.completed).to.equal true
+            done()
+
+      it "Creates an uncompleted todo when using down url", (done) ->
+        downTodo = undefined
+        request.post(baseURL + "/user/tasks/withDown/down").send(
+            type: "todo"
+            text: "testTitle"
+            complete: true
```

Comment:
So, I should have a create normal, a create to override, and the update test for todos below?